ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/c3-metal/private/provision/cucushift-installer-rehearse-gcp-ipi-c3-metal-private-provision-chain.yaml

```diff
@@ -0,0 +1,22 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-c3-metal-private-provision
+  steps: 
+  - ref: gcp-provision-c3-metal-network
+  - ref: ignition-bastionhost
+  - ref: gcp-provision-bastionhost
+  - ref: proxy-config-generate
+  - chain: ipi-conf-gcp
+  - ref: ipi-conf-gcp-control-plane-node-type
+  - ref: ipi-conf-gcp-confidential-computing
```

Comment:
Is the confidential step a MUST to run the c3 baremetal testing

Reply:
This step will insert `onHostMaintenance: Terminate` which is required for the c3 bare metal machine type.